### DISCUSSION TITLE
可以提供一个python版本的吗

### DISCUSSION THREAD

**Discussion by t3573393** [cloudwu/skynet#1492] [Q&A] [answered]
在服务器端开发的人，现在很多系统模块都使用python， 想能够复用其他的代码，最好能够有一个python的版本支持

**Comment by cloudwu** [comment]
python 有 GIL 问题，不可能如期望嵌入。

**Comment by cupen** [comment]
@t3573393 兄台有胆色用  `pypy-stm` 不？ 这玩意终于发布了，但目前只有 python2.7 的版本。

* https://pypy.org/download.html
* http://doc.pypy.org/en/latest/faq.html#does-pypy-have-a-gil-why
* http://doc.pypy.org/en/latest/stm.html

**Comment by cupen** [comment, marked as answer]
@t3573393  @cloudwu  
https://github.com/bytedance/pyskynet
